server/src/main/java/io/unitycatalog/server/persist/dao/CommitDAO.java

```diff
@@ -0,0 +1,79 @@
+package io.unitycatalog.server.persist.dao;
+
+import io.unitycatalog.server.model.Commit;
+import io.unitycatalog.server.model.CommitInfo;
+import jakarta.persistence.*;
+import java.util.Date;
+import java.util.UUID;
+import lombok.*;
+import org.hibernate.annotations.UuidGenerator;
+
+@Entity
+@Table(
+    name = "uc_commits",
+    uniqueConstraints = {
+      @UniqueConstraint(columnNames = {"metastore_id", "table_id", "commit_version"})
+    })
+// Lombok
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class CommitDAO {
```

Comment:
Will this class be used for just Delta commits or Iceberg commits as well?

Reply:
It is generic enough that it could be used for both but I haven't really looked at iceberg commits, do you have a reference?

Reply:
cc @ramesh-db this needs your attention.